docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
     restart: unless-stopped
     environment:
       - SIPI_EXTERNAL_PROTOCOL=http
-      - SIPI_EXTERNAL_HOSTNAME=sipi
+      - SIPI_EXTERNAL_HOSTNAME=localhost
```

Comment:
Question. This will work also on other environments? So uploading something on production will also return localhost as a hostname and will this address work?

Reply:
this is only the config for the local dev environment. Prod and Test have their own configuration.

Reply:
I've checked locally and works as expected. Thanks.